sql/2020/07_SEO/10_16_h1_length_percentiles_raw.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# 10_16: <h1> length
+SELECT
+  percentile,
+  client,
+  APPROX_QUANTILES(LENGTH(h1), 1000)[OFFSET(percentile * 10)] AS h1_length
+FROM (
+  SELECT
+    client,
+    REGEXP_EXTRACT(body, '(?i)<h1>([^(</h1>)]*)</h1>') AS h1
```

Comment:
Is the existing [`seo-titles`](https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js#L174) custom metric sufficient? Or if you need the h1 specifically, I'd propose:

```js
Array.from(document.querySelectorAll('h1')).map(e => e.innerText.replace(/\s+/g, ' '))
```

This accounts for multiple possible h1s per page.

Reply:
I've added a heading property to gather what we want this year. I may include some of those old metrics if they want them.

I noticed some of the SEO properties were used by other chapters. So I suspect it's not safe to alter them. I've deliberately left them alone and created new ones for what I want.